src/csharp/Intel/Iced.UnitTests/Intel/AssemblerTests/AssemblerTests64.cs

```diff
@@ -97,6 +97,20 @@ public sealed partial class AssemblerTests64 {
 		}
 #endif
 
+
+		[Fact]
+		public void TestDeclareData_db_array() {
+			Assert.Throws<ArgumentNullException>(() =>
+				TestAssemblerDeclareData(c => c.db(null), new byte[] { })
+			);
+			TestAssemblerDeclareData(c => c.db(new byte[] { (byte)1 }), new byte[] { (byte)1 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16 }), new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16, (byte)17 }), new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16, (byte)17 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16, (byte)17, (byte)18, (byte)19, (byte)20 }), new byte[] { (byte)1, (byte)2, (byte)3, (byte)4, (byte)5, (byte)6, (byte)7, (byte)8, (byte)9, (byte)10, (byte)11, (byte)12, (byte)13, (byte)14, (byte)15, (byte)16, (byte)17, (byte)18, (byte)19, (byte)20 });
```

Comment:
This one should have 32 elements.

There's also no empty array test.

Could you also remove `(byte)`, it's not needed and makes the lines very long.